test/IDE/print_ast_tc_decls.swift

```diff
@@ -396,8 +396,8 @@ extension d0110_ReadWriteProperties {
     }
     set(v) {}
   }
-// PASS_RW_PROP_GET_SET-NEXT:    {{^}}  static var extStaticProp: Int { get set }{{$}}
-// PASS_RW_PROP_NO_GET_SET-NEXT: {{^}}  static var extStaticProp: Int{{$}}
+// PASS_RW_PROP_GET_SET-NEXT:    {{^}}  static var extStaticProp: Int { get nonmutating set }{{$}}
+// PASS_RW_PROP_NO_GET_SET-NEXT: {{^}}  static var extStaticProp: Int { get nonmutating set }{{$}}
```

Comment:
@jrose-apple What do you think about fallout like this?

Reply:
That's incorrect; those *are* mutating sets.

Reply:
D'oh. You're right.